appsmithorg/appsmith
# fix: paste before copy resulted in value being undefined

## Description

If we paste before copying anything, `CopiedWidgetGroups` is undefined which results in some errors down the line.

Fixes #8941 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Manually:
1. Create an application in Incognito mode (just to make sure your copy history in indexed storage is empty)
2. Press `Cmd` + `V`

## Checklist:



## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>


    // Code coverage diff between base branch:release and head branch: fix/paste-before-copy 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | app/client/src/sagas/WidgetOperationSagas.tsx | 53.95 **(-0.02)** | 42.16 **(0.08)** | 53.19 **(0)** | 55.83 **(0.12)**
 :red_circle: | app/client/src/utils/storage.ts | 33.96 **(-0.22)** | 15.79 **(-0.88)** | 9.38 **(0)** | 26.05 **(-0.22)**</details>
